run-notification-trigger.js

```diff
@@ -0,0 +1,3 @@
+var notificationTrigger = require('./NotificationTrigger')
```

Comment:
@shrijan-siris As discussed earlier can you run locally through jest tests instead of creating dummy files to run triggers.